### DISCUSSION TITLE
getEntranceFee is returning 0

### DISCUSSION THREAD

**Discussion by naveenburathi** [smartcontractkit/full-blockchain-solidity-course-py#573] [Q&A] [answered]
Hlo everyone,
My problem isnt too complex I guess. I just cant figure out why I my getEntranceFee function returns 0 value which is different from the value from the video.

Here's my fund_and_withdraw script code.
<img width="470" alt="image" src="https://user-images.githubusercontent.com/64520563/146173741-943bf408-6647-44d4-ac96-2bb4713983a9.png">

Here's my getEntranceFee function from FundMe.sol code

<img width="759" alt="image" src="https://user-images.githubusercontent.com/64520563/146174022-7bb1d313-f359-401d-8bca-b01ee8542a2b.png">

And here's my ss of output
<img width="633" alt="image" src="https://user-images.githubusercontent.com/64520563/146174174-93efd260-0f58-48f1-917d-58ec9d538229.png">

**Comment by cromewar** [comment, marked as answer]
Well I reviewed everything and also run this on my machine and I wasn't able to reproduce it. May be this hints could help.

1. Make sure your getPrice() function is ok.
2. Also if you are testing this on local you must deploy mocks, otherwise it'll be always zero.
